ownCloud/Viewer/DisplayViewController.swift

```diff
@@ -40,9 +41,10 @@ protocol DisplayViewEditingDelegate: class {
 
 class DisplayViewController: UIViewController {
 
-	private let ICON_IMAGE_SIZE: CGSize = CGSize(width: 200.0, height: 200.0)
+	private let iconImageSize: CGSize = CGSize(width: 200.0, height: 200.0)
 
 	// MARK: - Configuration
+	weak var rootItem: OCItem!
```

Comment:
Same here… seems to be a left-over addition and can be removed. Also doesn't seem to be needed/used anywhere.